gpm_core/src/package.rs

```diff
@@ -27,7 +27,7 @@ pub struct PackageInformation {
 
     // optional values
     pub website_url: Option<String>,
-    pub dependencies: Vec<u8>,
+    pub dependencies: Vec<String>,
```

Comment:
The draft spec we currently have specifies the list of dependencies uses the packages IDs and not their names as the string implies.

But it's true that i don't understand how it would work with IDs. So don't consider this comment as blocking, but more as question. Do we follow the specs strictly or not?

Reply:
Id don't need to be integer.